OrbitQt/CMakeLists.txt

```diff
@@ -104,3 +104,26 @@ endif()
 set_target_properties(OrbitQt PROPERTIES OUTPUT_NAME "Orbit")
```

Comment:
please try to keep tests in the same commit with the class they are testing. Test provide important context for how particular class is expected to behave.

For example GgpInstase with the test could go in one commit, Model in next commit (with the test)...

Reply:
done